test/unit/action/unprocessable_entity_result_tests.rb

```diff
@@ -26,26 +31,17 @@ class InitTests < UnitTests
       controller1.instance_exec(subject, &subject.execute_block)
       assert_that(controller1.render_called_with)
         .equals(
-          json: "ERRORS JSON",
+          json:
+            {
+              field: "ERROR 1",
+              "nested[field]" => "ERROR 1",
+            },
           status: :unprocessable_entity
         )
-      assert_that(controller1.action_errors_json_called_with)
-        .equals(something: "ERROR")
     end
   end
 
   class FakeController
-    attr_reader :render_called_with, :action_errors_json_called_with
-
-    def render(**kargs)
-      @render_called_with = kargs
-    end
-
-    private
-
-    def action_errors_json(errors)
-      @action_errors_json_called_with = errors
-      "ERRORS JSON"
-    end
+    include FakeActionController
```

Comment:
Missing `require "test/support/fake_action_controller"` for this.

Reply:
yes - thanks!